octodns/provider/nsone.py

```diff
@@ -0,0 +1,189 @@
+#
+#
+#
+
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from logging import getLogger
+from nsone import NSONE
+from nsone.rest.errors import ResourceException
+
+from ..record import Record
+from .base import BaseProvider
+
+
+class NsOneProvider(BaseProvider):
```

Comment:
despite our python client still using our older `nsone` moniker, we were hoping you could shift this to be `NS1Provider` and in general use `ns1` instead of `nsone`, which is what we've rebranded to since the original python client came out.